docs/src/user-guide/command-line-interface.md

```diff
@@ -491,37 +525,56 @@ npx eslint --report-unused-disable-directives file.js
 
-**Note:** Autofixed files are not placed in the cache. Subsequent linting that does not trigger an autofix will place it in the cache.
+##### `--cache` example
+
+```shell
+npx eslint --cache file.js
+```
 
 #### `--cache-file`
 
-Path to the cache file. If no file is specified, `.eslintcache` is used. The file is created in the directory where the `eslint` command is executed. **Deprecated**: Use `--cache-location` instead.
+**Deprecated**: Use `--cache-location` instead.
+
+Path to the cache file. If none specified `.eslintcache` is used. The file is created in the directory where the `eslint` command is executed.
 
 #### `--cache-location`
 
-Path to the cache location. Can be a file or a directory. If no location is specified, `.eslintcache` is used. In that case, the file is created in the directory where the `eslint` command is executed.
+Specify the path to the cache location. Can be a file or a directory.
 
-If a directory is specified, a cache file is created inside the specified folder. The name of the file is based on the hash of the current working directory (CWD). e.g.: `.cache_hashOfCWD`
+* **Argument Type**: String. Path to file or directory. If a directory is specified, a cache file is created inside the specified folder. The name of the file is based on the hash of the current working directory, e.g.: `.cache_hashOfCWD`.
+* **Multiple Arguments**: No
+* **Default Value**: If no location is specified, `.eslintcache` is used. The file is created in the directory where the `eslint` command is executed.
 
-**Important note:** If the directory for the cache does not exist make sure you add a trailing `/` on \*nix systems or `\` in windows. Otherwise the path is assumed to be a file.
+If the directory for the cache does not exist make sure you add a trailing `/` on \*nix systems or `\` on Windows. Otherwise the path is assumed to be a file.
 
-Example:
+##### `--cache-location` example
 
 ```shell
 npx eslint "src/**/*.js" --cache --cache-location "/Users/user/.eslintcache/"
 ```
 
 #### `--cache-strategy`
 
-Strategy for the cache to use for detecting changed files. Can be either `metadata` or `content`. If no strategy is specified, `metadata` is used.
+Strategy for the cache to use for detecting changed files.
 
-The `content` strategy can be useful in cases where the modification time of your files changes even if their contents have not. For example, this can happen during git operations like git clone because git does not track file modification time.
+* **Argument Type**: String. One of the following values:
+  1. `metadata`
+  1. `content`
+* **Multiple Arguments**: No
+* **Default Value**: `metadata`
 
-Example:
+The `content` strategy can be useful in cases where the modification time of your files change even if their contents have not. For example, this can happen during git operations like `git clone` because git does not track file modification time.
```

Comment:
```suggestion
The `content` strategy can be useful in cases where the modification time of your files changes even if their contents have not. For example, this can happen during git operations like `git clone` because git does not track file modification time.
```